pkg/api/steve/supportconfigs/generator.go

```diff
@@ -54,12 +57,23 @@ func (h *GeneratorHandler) ServeHTTP(writer http.ResponseWriter, request *http.R
 		util.ReturnHTTPError(writer, request, http.StatusForbidden, "forbidden")
 		return
 	}
+	_, err = h.adapterUtil.GetRelease()
+	if err != nil {
+		if errors.Is(err, cspadapter.ErrNotFound) {
+			// if the adapter is not installed, return a 501 so the user knows to install the adapter
+			util.ReturnHTTPError(writer, request, http.StatusNotImplemented, "csp-adapter must be installed in order to to generate supportconfigs")
+			return
+		}
+		logrus.Warnf("[%s] Error when attempting to determine if adapter is installed, %s", logPrefix, err)
+		util.ReturnHTTPError(writer, request, http.StatusInternalServerError, "internal server error")
```

Comment:
Nit: for the last argument, it'd be a good practice to return the text for the HTTP status code using the `StatusText(code int) string` function from the `net/http` package.

Reply:
Good call, Change made.